src/opensearch.py

```diff
@@ -147,30 +147,24 @@ def start(self):
         except OpenSearchCmdError:
             raise OpenSearchStartError()
 
-        retries = 0
-        while not self.is_started() and retries < 3:
-            time.sleep(2)
-            retries += 1
+        start = datetime.now()
+        while not self.is_started() and (datetime.now() - start).seconds < 45:
+            time.sleep(3)
         else:
-            raise OpenSearchStartError()
+            raise OpenSearchStartTimeoutError()
 
     @override
     def _stop_service(self):
         """Stop opensearch."""
-        self._run_cmd(
-            "setpriv",
-            "--clear-groups --reuid ubuntu --regid ubuntu -- sudo systemctl stop opensearch.service",
```

Comment:
Because we're stopping the service. And we *can* stop it without assuming the `ubuntu` user. As opposed to when we want to start it as the `ubuntu` user